egs/swbd/s5c/local/reverberate/prep_sim_rirs.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to simulate room impulse responses (RIRs)
+
+sampling_rate=8000
```

Comment:
This is a temporary dependency. If these scripts are found to be useful we
will translate the necessary scripts to python (of course with a
scipy/numpy dependency).

On Wed, Apr 6, 2016 at 10:40 PM, Daniel Povey <notifications@github.com>
wrote:

> In egs/swbd/s5c/local/reverberate/prep_sim_rirs.sh
> <https://github.com/kaldi-asr/kaldi/pull/667#discussion_r58812176>:
>
> > @@ -0,0 +1,85 @@
> > +#!/bin/bash
> > +# Copyright 2016  Tom Ko
> > +# Apache 2.0
> > +# script to simulate room impulse responses (RIRs)
> > +
> > +sampling_rate=8000
>
> Are you talking about whether matlab itself is installed?
> Definitely there should be a check for that.
> I'm very unhappy with having a dependency on matlab at all-- many people
> can't afford it. But I suppose it won't be very easy to get rid of this
> dependency. Even ensuring octave compatibility would be better than relying
> on matlab itself.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/667/files/1683e258c9f571271668ab6e1d2b0898e7b7ed33#r58812176>
>